packages/core/echo/echo-db2/src/object.ts

```diff
@@ -76,24 +80,34 @@ export class EchoObjectBase {
       }
     }
 
-    return new Proxy(this, {
-      get: (target, property, receiver) => {
-        if (!isValidKey(property)) {
-          return Reflect.get(target, property, receiver);
-        }
+    return this._createProxy(this);
+  }
 
-        return this._get(property as string);
-      },
+  // TODO(burdon): Document.
+  get [Symbol.toStringTag]() {
+    return this[unproxy]?._schemaType?.name ?? 'EchoObject';
+  }
 
-      set: (target, property, value, receiver) => {
-        if (!isValidKey(property)) {
-          return Reflect.set(target, property, value, receiver);
+  /**
+   * Convert to JSON object.
+   */
+  json() {
```

Comment:
rename to `toJSON` to follow the convention: https://javascript.info/json

Reply:
Done and added tests.